.github/ISSUE_TEMPLATE/new-feature-request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: New Feature request
+about: Request a new feature for Skin
+title: "[NEW FEATURE]"
+labels: 'type: feature'
+assignees: ''
+
+---
+
+**Describe the feature/module in detail**
+<!-- A description of how the feature/module is intended to work. -->
+
+**Screenshots**
+<!-- Add screenshots of the feature/module. -->
+
+**Figma link**
+<!-- Add a link to the Figma page for the feature/module. -->
+
+**Prerequisites**
+
+- [ ] Successful design review with Design System Team.
+- [ ] Supports dark mode.
+- [ ] Supports responsive design.
+- [ ] Tokens have been provided (or already exist).
+- [ ] Successful accessibility review with Core A11Y Team.
+- [ ] Successful frontend review with eBayUI Team.
+
+Note: A "successful review" means those teams give their blessing for it to proceed to engineering.
```

Comment:
Maybe put this in a comment too? I'm not sure if we need to see this message at the end of every issue

Reply:
Actually, I think it might be cool to put this in a note card

```
> **Note**
> A "successful review" means those teams give their blessing for it to proceed to engineering.
```

> **Note**
> A "successful review" means those teams give their blessing for it to proceed to engineering.

Reply:
You can use GitHub's suggestion feature to make this easy for PR owner to commit.

```suggestion
> **Note**
> A "successful review" means those teams give their blessing for it to proceed to engineering.
```